src/Web/Pages/Profile/EditInterests.razor

```diff
@@ -171,70 +158,41 @@
 			if (ExpertiseAsList.Contains(i.Trim().ToLower()) == false)
 			{
 				ExpertiseAsList.Add(i.Trim().ToLower());
+				ShowSaveMessage = true;
 			}
 		}
-
-		// Reset the search term and clear-out suggestions.
-		SearchTerm = "";
-		await SearchTermChanged(SearchTerm);		
-	}
-
-	private async Task SearchTermListenForEnter(KeyboardEventArgs e)
-	{
-		if (e.Code == "Enter" || e.Code == "NumpadEnter")
-		{
-			await AddInterest(SearchTerm);
-		}
+		StateHasChanged();// Needed because this is invoked by a child component
 	}
 
 	private async Task SaveChanges()
 	{
 		try
 		{
-			Person result;
-
-			// Make an in-memory copy of Person to PUT to the server.
-			var objToPut = JsonConvert.DeserializeObject<Person>(JsonConvert.SerializeObject(UpdatedPerson, Json.JsonSerializerSettings), Json.JsonSerializerSettings);
-			// Update objToPut's Expertise
-			objToPut.Expertise = string.Join(",", ExpertiseAsList);
-
-			var url = $"/people/{UpdatedPerson.Id}";
-
-			using (var apiClient = Api())
+			using(Loading())
 			{
-				var reqJson = JsonConvert.SerializeObject(objToPut, Json.JsonSerializerSettings);
-				var reqContent = new StringContent(reqJson, System.Text.Encoding.UTF8, "application/json");
-				var response = await apiClient.PutAsync(url, reqContent);
+				// Make an in-memory copy of Person to PUT to the server.
+				var objToPut = JsonConvert.DeserializeObject<Person>(JsonConvert.SerializeObject(UpdatedPerson, Json.JsonSerializerSettings), Json.JsonSerializerSettings);
```

Comment:
I don't think this is the first time we've had to pull this off. We might be able to put this up into pageBase or something

Reply:
We don't need to do it all the time, but there are cases where we don't want to fiddle withe the original object.  Maybe add a `PageBase.MakeCopy(Entity source)` method?

Reply:
Added DeepCopy to PageBase. Resolved

Reply:
Implemented in 2ec6572 and 25866fa